Improve how we operate by linking to recent changes and information (#8382)
istio/istio.io: content/en/about/community/join/index.md

{{< /community_item >}}

{{< community_item logo="./group.svg" alt="Working Groups" >}}
If you'd like to contribute to the Istio project, please read our

> [code of conduct agreement](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct) and

Reviewer: Looking at the preview, I think I agree with @ericvn's comment about the word "agreement". It's not necessary.
```suggestion
[code of conduct](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct) and
```

[contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md), and
consider participating in our [working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
{{< /community_item >}}

{{< community_item logo="./governance.svg" alt="Governance" >}}
If you would like to know more about Istio governance, please learn [how we do things](https://github.com/istio/community/blob/master/README.md)

> as a project, and read in more detail about the [Istio Steering Committee](https://github.com/istio/community/tree/master/steering)
> and the [Istio Technical Oversight Committee](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md).

Reviewer 2: ```suggestion
as a project, and read in more detail about the Istio [Steering Committee](https://github.com/istio/community/tree/master/steering)
and [Technical Oversight Committee](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md).
```

perhaps?

{{< /community_item >}}

{{< community_item logo="./group.svg" alt="Working Groups" >}}

> If you'd like to contribute to the Istio project, please read about our

Reviewer: ```suggestion
If you'd like to contribute to the Istio project, please read our
```

> [code of conduct](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct), our

Reviewer: ```suggestion
[code of conduct aggreement](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct) and
```

Reviewer 3: Not sure I like `agreement` (misspelled above).

Reviewer: Thanks, I fixed the spelling in the suggested change, above. Don't want to introduce another linter error :)

Reviewer: I suppose "agreement" is optional, since "code of conduct" is itself a noun that can stand on its own.

[contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md), and
consider participating in our [working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
{{< /community_item >}}

> {{< community_item logo="./group.svg" alt="Governance" >}}

Reviewer: Change icon as suggested in https://github.com/istio/istio.io/pull/8382#issuecomment-718821693

> If you would like to learn more about Istio governance, please learn [how we do things](https://github.com/istio/community/blob/master/README.md)

Reviewer: ```suggestion
If you would like to know more about Istio governance, please learn [how we do things](https://github.com/istio/community/blob/master/README.md)
```

> as a project, and in more detail learn about the [Istio Steering Committee information](https://github.com/istio/community/tree/master/steering)

Reviewer: ```suggestion
as a project, and read in more detail about the [Istio Steering Committee](https://github.com/istio/community/tree/master/steering)
```
